File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
    int frS[28],frT[28];
public:
    bool isAnagram(string s, string t) {
        
       if(s.length()!=t.length())return false;
        
        for(int i=0;i<s.length();++i){
            frS[s[i]-'a']++;
            frT[t[i]-'a']++;
        }
        
        for(int i=0;i<t.length();++i){
            if(frS[t[i]-'a']!=frT[t[i]-'a'])return false;
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.cs
public class Solution {
    public bool IsAnagram(string s, string t) {
        if(t.Length!=s.Length)return false;
        int n=s.Length;
        var frequencyS=new int[28];
        var frequencyT=new int[28];
        for(int i=0;i<n;++i){
          frequencyS[s[i]-'a']++;
          frequencyT[t[i]-'a']++;
        }
          for(int i=0;i<n;++i){
          if(frequencyS[t[i]-'a']!=frequencyT[t[i]-'a'])
              return false;
        }
        return true;
    }
}